addons/point_of_sale/static/src/app/utils/html-to-image.js

```diff
@@ -209,8 +209,11 @@ function canvasToBlob(canvas, options = {}) {
 function createImage(url) {
     return new Promise((resolve, reject) => {
         const img = new Image();
-        img.decode = () => resolve(img);
-        img.onload = () => resolve(img);
+        img.onload = () => {
+            img.decode().then(() => {
+                requestAnimationFrame(() => resolve(img));
+            });
+        };
```

Comment:
What do you think about contributing the fix to the source? https://github.com/bubkoo/html-to-image

Reply:
Sure, why not! Will do so shortly

Reply:
It might take time though. If they are not responsive, what we can do is to keep a fork for us and put there the fixes.

Reply:
Here's the PR on the source:https://github.com/bubkoo/html-to-image/pull/478